org.mixedrealitytoolkit.spatialmanipulation/Solvers/HandConstraintPalmUp.cs

```diff
@@ -296,14 +296,16 @@ private bool IsUserGazeMeetingThresholdRequirements(XRNode hand)
                 #pragma warning restore CS0618
                 else if (TrackedPoseDriverLookup != null)
                 {
-                    InputTrackingState gazeTrackingStateInput = (InputTrackingState)TrackedPoseDriverLookup.GazeTrackedPoseDriver.trackingStateInput.action.ReadValue<int>();
-                    if (TrackedPoseDriverLookup.GazeTrackedPoseDriver != null &&
-                        gazeTrackingStateInput.HasFlag(InputTrackingState.Position) &&
+                    InputTrackingState gazeTrackingStateInput = TrackedPoseDriverLookup.GazeTrackedPoseDriver != null ?
+                        (InputTrackingState)(TrackedPoseDriverLookup.GazeTrackedPoseDriver.trackingStateInput.action?.ReadValue<int>() ?? default) :
```

Comment:
Do we want to explicitly set this to `None` in the null case?

```suggestion
                        (InputTrackingState)(TrackedPoseDriverLookup.GazeTrackedPoseDriver.trackingStateInput.action?.ReadValue<int>() ?? InputTrackingState.None) :
```

Reply:
Ohh or does this run into that not-yet-available-in-Unity feature where each side of the `??` is returning different types?

Reply:
Yeah, it was complaining if I tried that explicit value, but default should just be a 0 int, which is effectively the same thing and gets cast as None.

Adam has fixed this section of code in his larger PR, so this likely won't be used anyway.